pkg/cli/admin/release/info.go

```diff
@@ -91,9 +91,8 @@ func NewInfo(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Comm
 			a tag when verifying an image is recommended since it ensures an attacker cannot trick you
 			into installing an older, potentially vulnerable version.
 
-			The --bugs and --changelog flags will use git to clone the source of the release and display
-			the code changes that occurred between the two release arguments. This operation is slow
-			and requires sufficient disk space on the selected drive to clone all repositories.
+			The --bugs and --changelog flags will use git to clone the git history of the release and display
+			the code changes that occurred between the two release arguments.
```

Comment:
nit: What about still mentioning sufficient disk space as current state, since we may still perform full clone. WDYT?

Reply:
I've added back the note on speed and disk space.